src/recommendations/implementation.md

```diff
@@ -7,7 +7,7 @@ ee_only: True
 To help you implement product recommendations on your storefront, use the following workflow:
 
 1. **Deploy data collection to production**
-   -  **Why** - The more data you collect the more accurate your recommendations will be. As soon as you start collecting data, you can pull that data into your non-production environment and experiment with different recommendation types on different pages.
+   -  **Why** - Product Recommendations are made up of two main data sources: [catalog and behavioral]({{ page.baseurl }}/recommendations/product-recs.html#types-of-data). By deploying data collection to production, you are enabling a JavaScript to collect the behavioral data. The more data you collect the more accurate your recommendations will be. As soon as you start collecting data, you can pull that data into your non-production environment and experiment with different recommendation types on different pages.
```

Comment:
`enabling a JavaScript to collect the behavioral data`

Should we say "enabled a JavaScript application" or something like that? "Enabling a JavaScript sounds a little awkward.

Reply:
Oops...yes! I'll clean that up.